TFNetworkRecLayer.py

```diff
@@ -1595,7 +1596,7 @@ def maybe_transform(layer):
           for k in self._initial_extra_outputs:
             args.update({"%s.extra.%s" % (k, k2): v for (k2, v) in self.net.layers[k].rec_vars_outputs.items()})
             args.update({"prev:%s.extra.%s" % (k, k2): v for (k2, v) in prev_extra[k].items()})
-          res = (identity_with_debug_log(out=self._debug_out, x=res[0], args=args),) + res[1:]
+          res = (identity_with_debug_log(out=self._debug_out, source=res[0], args=args),) + res[1:]
```

Comment:
identity_with_debug_log has the arg name `x`, not `source`.

Reply:
Automatic renaming... :D